xds/internal/resolver/serviceconfig.go

```diff
@@ -381,17 +371,19 @@ func (r *xdsResolver) newConfigSelector(su serviceUpdate) (*configSelector, erro
 	}
 
 	for i, rt := range su.virtualHost.Routes {
+		clusters := newWRR()
 		if rt.ClusterSpecifierPlugin != "" {
+			clusters.Add(&routeCluster{
+				name:                     "cluster:" + rt.ClusterSpecifierPlugin,
```

Comment:
Shouldn't this be `"cluster_specifier_plugin:" + rt.ClusterSpecifierPlugin`?

Please use a local to hold that instead of repeating it 4 times.  Also, global consts for `"cluster:"` and `"cluster_specifier_plugin:"` would be a good idea to avoid any chance of a typo in one usage.

Reply:
Yeah, good catch. Added global consts and also a local var for both branches.

Reply:
Also, the comment isn't showing up on this PR, but I refactored the cluster initialization into another function. It made it much cleaner.